<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="glInterface">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item row="2" column="2">
       <widget class="QListWidget" name="lwRight"/>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pbLeftSort">
        <property name="text">
         <string>Sort (in ascending order)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QVBoxLayout" name="vlButtons">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="pbToRight">
          <property name="font">
           <font>
            <family>Segoe UI Historic</family>
            <pointsize>36</pointsize>
           </font>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbToLeft">
          <property name="font">
           <font>
            <family>Segoe UI Historic</family>
            <pointsize>36</pointsize>
           </font>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pbRightSort">
        <property name="text">
         <string>Sort (in descending order)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QListWidget" name="lwLeft"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
